vividus-tests/src/main/resources/story/integration/PageStepsTests.story

```diff
@@ -21,6 +21,15 @@ When I change context to element located by `cssSelector(img)`
 Then `${context-height}` is < `100`
 Then `${context-width}`  is < `100`
 
+
+Scenario: Verify step: When I refresh page; Verify step: Then page title $comparisonRule `$text`
+Given I am on page with URL `${vividus-test-site-url}`
+When I refresh page
+Then page title is equal to `Vividus Test Site`
```

Comment:
I believe we have very similar validations in `SetContextStepsTests.story`, please, make sure no new duplicate tests are added